docs/quality-assurance/README.md

```diff
@@ -2,27 +2,26 @@
 
 These documents detail how we work as a Test Community at UKHO. We aim to provide a framework for our testers, developers and community engineers to enable them to deliver testing that supports our desire to build and release good software quickly.
 
-## Content
+## Where to start
 
-* [Our Vision](test-vision.md)
-  * The principles that guide everything we do as a Test Community
-* [Our UKHO Quality Charter](ukho-quality-charter.md)
-  * The UKHO Quality Charter that delivery teams are expected to adhere to
-* [Our Testing Principles](testing-principles.md)
-  * The principles that guide our testing
-* [Our Test Strategy](test-strategy.md)
-  * Our over-arching testing strategy
-* [Our Test Automation Strategy](test-automation-strategy.md)
-  * The guidelines for using Test Automation
-* [Our Test Standards](test-code-standards.md)
-  * How we plan, write and organise our tests
-* [Our Test Policy](test-policy.md)
-  * UKHO overall testing policy
-* [Our Testers](test-profiles.md)
-  * Our test roles and their responsibilities
-* [Our Open Source Repositories](test-repositories.md)
-  * The projects produced that support the way we work
-* [Our Test Tools](test-tooling.md)
-  * The tools we use to carry out our automated and manual testing
-* [BDD Reference Page](bdd.md)
-  * Information on BDD to aid our use at UKHO
+If you read nothing else, then read these three things:
```

Comment:
I like this but wonder if it could be re-worded, maybe "These pages cover the key areas for our approach to testing" or similar...

Reply:
I'm thinking that this section is more of a promotion/highlights area. Those top three things could well change